content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -48,13 +48,7 @@ The steps for enabling debug mode in the Datadog Python Tracer depends on the ve
 
 2. To route logs to the console, for **Python 2 or Python 3** applications, configure `logging.basicConfig()` or use `DD_CALL_BASIC_CONFIG=true`.
 
-### Scenario 3: ddtrace version 0.x
```

Comment:
For support purposes, I think we should leave the 0.x instructions up here for a while, but maybe we can add a sentence that says `0.x is now end of life and we recommend upgrading` and remove it after a period of time?